docs/guide/front-end-assets.md

```diff
@@ -68,18 +73,55 @@ The output will **sing** before it **dances**. If the modules are instantiated i
+
+```js
+// modules/@apostrophecms/asset/index.js
+module.exports = {
+  options: {
+    es5: true
+  }
+};
+```
 
--  The `dev` npm script in `package.json` runs a project-level webpack build, which compiles `src/index.js` according to the rules in `webpack.config.js`.
--  `src/index.scss` (written in [Sass](https://sass-lang.com/)) is imported by `src/index.js`, allowing the browser to load one file for both CSS and JS.
--  At the end of the build, the bundled assets are written to `modules/asset/ui/public/sites.js`.
-- Any `.js` files in the `ui/public` folder of any module are automatically included in the frontend build.
-- Whenever code changes are made, `nodemon` automatically restarts this cycle and refresh the browser after a successful restart.
+This configures Apostrophe to compile two JavaScript bundles: one for Internet Explorer 11 users, and one for everyone else. The Internet Explorer 11 bundle contains code compiled with `babel` in order to add support for missing features, as well as "polyfills" for missing functions.
 
-None of those specifics are required for Apostrophe. You can follow whatever organizing pattern works best for you or your organization.
+That makes the bundle larger of course, so Apostrophe serves a separate, small and fast bundle to non-IE11 users so there is **no performance penalty** for them.
+
+::: note Internet Explorer 11 Limitations
+Apostrophe uses babel's `@babel/preset-env` module to provide as much support as possible for JavaScript language features, including promises, `async/await` and more. However, Apostrophe does not provide polyfills for all browser API features, and some, such as `Observable`, cannot be implemented for Internet Explorer 11. Apostrophe also does not attempt to fix the limitations of CSS in Internet Explorer 11. And of course, the most obvious limitation is that **Apostrophe's editing interface is not available in Internet Explorer 11.** You should test your sites thoroughly in all browsers you intend to support.
 :::
+
+## Using your own Webpack configuration
+
+Apostrophe's built-in `ui/src` feature is very handy, and sufficient for most sites. But if you wish to customize your Webpack configuration, for instance to use `jsx` files for React or `.vue` files for Vue, then you'll want to roll your own.
+
+The basic idea is that your Webpack build will produce just one output `.js` file, and possibly a `.css` file too. Then you can feed those into Apostrophe by creating a generic module, like `asset`, and configuring Webpack to write its output to `modules/asset/ui/public/index.js` and `modules/asset/ui/public/index.css`. `ui/public` is similar to `ui/src`, with one important difference: Apostrophe imports the code exactly "as-is."
```

Comment:
This is a dense paragraph if you don't already know how to do this. We should have code examples and a bit more in this and the next section. Maybe rather than try go get it right today you could create a ticket for me to massage it out.